CoreScriptsRoot/Modules/Chat.lua

```diff
@@ -2457,8 +2461,11 @@ local function CreateChat()
 					if InputService.VREnabled then
 						self.Settings.TextStrokeTransparency = 1
 						self:PrintVRWelcome()
-						local Panel3D = require(CoreGuiService:WaitForChild('RobloxGui').Modules.VR.Panel3D)
-						local panel = Panel3D.Get("Chat")
+						local RobloxGui = CoreGuiService:WaitForChild('RobloxGui')
```

Comment:
Didn't we already store RobloxGui in a variable at the top? Is it not possible to re-use that?